pages/platform/kubernetes-k8s/deploying-gpu-application/guide.en-asia.md

```diff
@@ -0,0 +1,344 @@
+
+![Select a flavor for your GPU node pool](images/create-a-nodepool-3.png){.thumbnail}
+
+In the next step, define the size of your GPU node pool.
+You can enable the `Autoscaling`{.action} feature.
+Define the minimum and maximum pool size in that case, 3 in minimum and 10 in maximum, for example.
+
+![Define a size and autoscaling for your GPU node pool](images/create-a-nodepool-4.png){.thumbnail}
+
+Then, choose `Hourly` billing mode for this GPU node pool.
+
+![Billing mode](images/create-a-nodepool-5.png){.thumbnail}
+
+The node pool creation is now in progress. It should be available within a few minutes in your OVHcloud Control Panel.
+
+When the Node Pool is in `OK` state, if you display Node pool labels you can verify that the new `gpu-node-pool`'s Nodes are correctly labelled.
+
+```bash
+kubectl get node --show-labels | grep "node.k8s.ovh/type=gpu"
+```
+
+![GPU nodes](images/create-a-nodepool-6.png){.thumbnail}
+
+### NVIDIA GPU Operator
+
+The [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) uses the [Operator Framework](https://github.com/operator-framework/getting-started) within Kubernetes to automate the management of all NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers (to enable CUDA), Kubernetes device plugin for GPUs, the NVIDIA Container Runtime, automatic node labelling, DCGM based monitoring and others.
+
+![NVIDIA GPU Kubernetes Operator](images/nvidia-gpu-operator.jpeg)
+
+If you are interested about the operator, feel free to read the [NVIDIA GPU operator official documentation](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/overview.html).
+
+
+### Installing the NVIDIA GPU Operator Helm chart
+
+For this tutorial we are using the [NVIDIA GPU Operator Helm chart](https://github.com/NVIDIA/gpu-operator/tree/master/deployments/gpu-operator){.external} found on [NVIDIA repository](https://github.com/nvidia/). The chart is [fully configurable](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/getting-started.html), but here we are using the default configuration, with only the minimal set of customization to make it work well on OVHcloud Managed Kubernetes Service.
+
+Add the NVIDIA Helm repository:
+
+```bash
+helm repo add nvidia https://nvidia.github.io/gpu-operator
+helm repo update
+```
+
+This will add the NVIDIA repository and update all of your repositories: 
+
+<pre class="console"><code>$ helm repo add nvidia https://nvidia.github.io/gpu-operator
+helm repo update
+"nvidia" has been added to your repositories
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "nvidia" chart repository
+[...]
+Update Complete. ⎈Happy Helming!⎈
+</code></pre>
+
+Install the GPU Operator in the `gpu-operator` namespace:
+
+```bash
+$ helm install gpu-operator nvidia/gpu-operator -n gpu-operator --create-namespace --wait
```

Comment:
Is it possible to remove any special character in front of "copy/paste" command line?

Reply:
And maybe it could be interesting to add a little  warning here.
We using the `latest` version of this Helm chart and it may not compatible with older release of k8s? (to be confirmed, I haven't tried this operator on my side ^^)

Reply:
the nvidia operator is correctly maintaned but yes I see what you are saying :-D